test/util.ts

```diff
@@ -64,3 +66,15 @@ export async function zimcheckAvailable() {
 export async function zimcheck(filePath: string) {
     return execa.command(`${zimcheckPath} ${filePath}`);
 }
+
+export async function convertWikicodeToHtml(wikicode: string, baseUrl: string): Promise<string> {
+    try {
+        const resp = await axios.post(`${baseUrl}api/rest_v1/transform/wikitext/to/html`,  {
+            wikitext: wikicode,
+            body_only: true,
+        })
+        return resp.data;
+    } catch (err){
+        logger.log(`Got err during conversion of wikicode to HTML due to ${err}`);
```

Comment:
* `error` not `err`.
* the whole tst shoudl fail here if the HTML is not retrievable properly
* I don't think the temporary variable `resp` is really necessary just return directly the data.